package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Simple plugin for Leaflet that adds fullscreen button to your maps.",
   "main": "Control.FullScreen.js",
   "scripts": {
-    "lint": "eslint --config .eslintrc Control.FullScreen.js",
+    "lint": "npx eslint . && stylelint **/*.css && prettier --check .",
```

Comment:
I am so sorry, I was wrong: npx is only required, when starting eslint from the shell. Somehow I mixed things up.

Reply:
Absolutely no problem! 🙂 I just removed npx.